programs/ssl/ssl_server2.c

```diff
@@ -3742,6 +3745,13 @@ int main( int argc, char *argv[] )
     }
 #endif /* MBEDTLS_SSL_DTLS_CONNECTION_ID */
 
+#if defined(MBEDTLS_MEMORY_DEBUG)
+    mbedtls_memory_buffer_alloc_cur_get(&current_heap_memory, &heap_blocks);
```

Comment:
Style: Need spaces after `(` and before `)`, here and other places in commit "Add an acceptance test for memory usage after handshake"